lib/binary_to_decimal.rb

```diff
@@ -4,6 +4,15 @@
 # The least significant bit is at index 7.
 # Calculate  and return the decimal value for this binary number using
 # the algorithm you devised in class.
+
 def binary_to_decimal(binary_array)
-  raise NotImplementedError
+  exponent_array = [2**7, 2**6, 2**5, 2**4, 2**3, 2**2, 2**1, 2**0]
```

Comment:
This works, but you've hardcoded for specific powers of 2.  Can you think of a way to make this more flexible?